src/open_inwoner/templates/pages/profile/contacts/list.html

```diff
@@ -103,8 +103,10 @@ <h2 class="h2">{% trans "U bent toegevoegd als contactpersoon" %}</h2>
                         <td class="table__item">{{invite.invitee_email}}</td>
                         <td class="table__item"></td>
                         <td class="table__item">{% icon "close" extra_classes="icon icon--danger" %}</td>
-                        <td class="table__item">{% button text=_('Stuur bericht') icon="message" icon_position="before" icon_outlined=True transparent=True disabled=True %}</td>
-                        <td class="table__item">{% button icon="settings" icon_position="before" text="" hide_text=True transparent=True disabled=True %}<td>
+			{% if inbox_page_is_published %}
+			    <td class="table__item">{% button text=_('Stuur bericht') icon="message" icon_position="before" icon_outlined=True transparent=True disabled=True %}</td>
```

Comment:
Thanks for pointing this out. It seems ugly to warp the entire `td` element in an `if/else` because of `colspan` (though it works). Is there a more elegant solution?

Reply:
We already use a lot the if statements in the templates and I don't think it's a problem to do it here as well. We can do something like this instead of wrapping the entire element:
`<td class="table__item"  {% if inbox_page_is_published %}colspan="2" {% else %}colspan="1"{% endif %}`

Another solution would be to add a variable to the `get_context_data` method of the view. It's up to you!

Reply:
Thanks for the suggestion!